### PR TITLE
Make some changes to README.md

### PR DESCRIPTION
**Pull request by DoNotSpamPls**
- Edited the "Current Containers" section so it doesn't have to be always updated
- Fixed a typo
- Bumped the copyright year.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,14 +2,12 @@
...
+Our goal here is to maintain images that can run a wide variety of games in individual branches. For example, the [`java` branch](https://github.com/Pterodactyl/Containers/tree/java) is a standardized Docker container running on Alpine Linux that should support most all Minecraft based games, as well as java based games. We install all of the required dependencies that we ae aware of, but if you notice one missing, we welcome you to either raise an issue in here, or submit a PR to add it.
 
 The `entrypoint.sh` files **should not do any application installing**, and exist solely to parse startup commands passed to the container, and then run them, with a pause at the beginning to allow the Daemon time to boot and attach to the container.
 
 ## Current Containers
-[`java`](https://github.com/Pterodactyl/Containers/tree/java)
-
-[`source`](https://github.com/Pterodactyl/Containers/tree/source)
+Every branch, except master, is a different container. For a current list of containers, check the branch list [here](https://github.com/Pterodactyl/Containers/branches/active)
```
[the remark this answers is not in the document]
Comment (DoNotSpamPls) [reply README.md]: Gotcha.

**Comment by DoNotSpamPls** [comment]
done 👀

**Comment by schrej** [comment]
Only Dane can merge, he'll do so when he sees this ;)

File: README.md [hunk 2]
```diff
@@ -2,14 +2,12 @@
...
+Our goal here is to maintain images that can run a wide variety of games in individual branches. For example, the [`java` branch](https://github.com/Pterodactyl/Containers/tree/java) is a standardized Docker container running on Alpine Linux that should support most all Minecraft based games, as well as java based games. We install all of the required dependencies that we ae aware of, but if you notice one missing, we welcome you to either raise an issue in here, or submit a PR to add it.
 
 The `entrypoint.sh` files **should not do any application installing**, and exist solely to parse startup commands passed to the container, and then run them, with a pause at the beginning to allow the Daemon time to boot and attach to the container.
 
 ## Current Containers
-[`java`](https://github.com/Pterodactyl/Containers/tree/java)
-
-[`source`](https://github.com/Pterodactyl/Containers/tree/source)
+Every branch, except master, is a different container. For a current list of containers, check the branch list [here](https://github.com/Pterodactyl/Containers/branches/active).
```
Comment (DaneEveritt) [inline README.md]: They are images, not containers. :)

Comment (schrej) [reply README.md]: Well, you could rename the whole repo while you're at it then :P

**Comment by DoNotSpamPls** [comment]
👍

**Comment by DoNotSpamPls** [comment]
bump

**Pull request merged**